composer/tools/requirements.txt

```diff
@@ -1,3 +1,6 @@
+cryptography==2.3.1
 google-api-python-client==1.6.4
 google-auth==1.5.1
 google-cloud-storage==1.11.0
+kuberneteskubernetes==7.0.0
+mysql-connector==2.1.6
```

Comment:
Looks like [`mysql-connector` is an old fork](https://github.com/sanpingz/mysql-connector). The most recent version is `mysql-connector-python` ([package](https://pypi.org/project/mysql-connector-python/), [repo](https://github.com/mysql/mysql-connector-python)).